nix/default.nix

```diff
@@ -81,12 +82,11 @@ in
           udis86
           wayland
           wayland-protocols
-          wayland-scanner
           pciutils
           (wlroots.override {inherit enableXWayland hidpiXWayland nvidiaPatches;})
         ]
-        ++ lib.optionals enableXWayland [libxcb xcbutilwm xwayland]
-        ++ lib.optionals withSystemd [systemd];
+        ++ lib.optionals enableXWayland [ libxcb xcbutilwm xwayland ]
+        ++ lib.optionals withSystemd [ systemd ];
```

Comment:
I believe the default formatter for Hyprland is Alejandra, please run `nix fmt` within the repository.

Reply:
Alejandra did the opposite of what I did，it removed those spaces.

```diff
--- a/nix/default.nix
+++ b/nix/default.nix
@@ -123,7 +123,7 @@ in
         ln -s ${wlroots}/include/wlr $dev/include/hyprland/wlroots
         ${lib.optionalString wrapRuntimeDeps ''
           wrapProgram $out/bin/Hyprland \
-            --suffix PATH : ${lib.makeBinPath [ binutils pciutils ]}
+            --suffix PATH : ${lib.makeBinPath [binutils pciutils]}
         ''}
       '';

```

Reply:
Correct. Those spaces were a part of the formatting.